[2025-02-05T14:31:06.380+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-05T14:31:06.400+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_dbt-model_sales.model_sales.ad_hoc_summary_vendas_run scheduled__2025-02-04T04:00:00+00:00 [queued]>
[2025-02-05T14:31:06.412+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_dbt-model_sales.model_sales.ad_hoc_summary_vendas_run scheduled__2025-02-04T04:00:00+00:00 [queued]>
[2025-02-05T14:31:06.414+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-02-05T14:31:06.428+0000] {taskinstance.py:2330} INFO - Executing <Task(DbtRunLocalOperator): model_sales.ad_hoc_summary_vendas_run> on 2025-02-04 04:00:00+00:00
[2025-02-05T14:31:06.435+0000] {standard_task_runner.py:63} INFO - Started process 170 to run task
[2025-02-05T14:31:06.440+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_dbt-model_sales', 'model_sales.ad_hoc_summary_vendas_run', 'scheduled__2025-02-04T04:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/dag_dbt-model_sales.py', '--cfg-path', '/tmp/tmpvy6yipls']
[2025-02-05T14:31:06.443+0000] {standard_task_runner.py:91} INFO - Job 8: Subtask model_sales.ad_hoc_summary_vendas_run
[2025-02-05T14:31:06.507+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_dbt-model_sales.model_sales.ad_hoc_summary_vendas_run scheduled__2025-02-04T04:00:00+00:00 [running]> on host eb17fa5f9c23
[2025-02-05T14:31:06.602+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_dbt-model_sales' AIRFLOW_CTX_TASK_ID='model_sales.ad_hoc_summary_vendas_run' AIRFLOW_CTX_EXECUTION_DATE='2025-02-04T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-04T04:00:00+00:00'
[2025-02-05T14:31:06.604+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-05T14:31:09.758+0000] {local.py:220} INFO - dbtRunner is available. Using dbtRunner for invoking dbt.
[2025-02-05T14:31:09.760+0000] {local.py:436} INFO - Cloning project to writable temp directory /tmp/tmp_ignq2zg from /usr/local/airflow/dags/dbt/localiza_dw
[2025-02-05T14:31:09.771+0000] {local.py:447} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/dag_dbt-model_sales__model_sales/target/partial_parse.msgpack
[2025-02-05T14:31:09.785+0000] {config.py:348} INFO - Profile caching is enable.
[2025-02-05T14:31:09.801+0000] {base.py:84} INFO - Using connection ID '***' for task execution.
[2025-02-05T14:31:09.803+0000] {config.py:328} INFO - Profile found in cache using profile: /tmp/cosmos/profile/47873256e3df2d6067f929148b3671caee72f33396318ef25aae33418849eddf/profiles.yml.
[2025-02-05T14:31:09.804+0000] {local.py:408} INFO - Trying to run dbtRunner with:
 ['run', '--vars', '{"PROD": P003,                                 "LIMITE_QUANTIDADE": 500,                                 "DATA_INICIO": 1999-01-01,                                 "DATA_FIM": 2025-02-05}', '--models', 'ad_hoc_summary_vendas', '--project-dir', '/tmp/tmp_ignq2zg', '--profiles-dir', '/tmp/cosmos/profile/47873256e3df2d6067f929148b3671caee72f33396318ef25aae33418849eddf', '--profile', '***', '--target', 'prod']
 in /tmp/tmp_ignq2zg
[2025-02-05T14:31:09.846+0000] {logging_mixin.py:188} INFO - 14:31:09  Running with dbt=1.6.1
[2025-02-05T14:31:10.053+0000] {logging_mixin.py:188} INFO - 14:31:10  Registered adapter: postgres=1.6.1
[2025-02-05T14:31:10.901+0000] {logging_mixin.py:188} INFO - 14:31:10  Found 4 models, 1 snapshot, 4 seeds, 5 tests, 3 sources, 0 exposures, 0 metrics, 349 macros, 0 groups, 0 semantic models
[2025-02-05T14:31:10.905+0000] {logging_mixin.py:188} INFO - 14:31:10
[2025-02-05T14:31:10.999+0000] {logging_mixin.py:188} INFO - 14:31:10  Concurrency: 1 threads (target='prod')
[2025-02-05T14:31:11.001+0000] {logging_mixin.py:188} INFO - 14:31:11
[2025-02-05T14:31:11.005+0000] {logging_mixin.py:188} INFO - 14:31:11  1 of 1 START sql table model dm_sales.ad_hoc_summary_vendas .................... [RUN]
[2025-02-05T14:31:11.134+0000] {logging_mixin.py:188} INFO - 14:31:11  1 of 1 OK created sql table model dm_sales.ad_hoc_summary_vendas ............... [SELECT 5 in 0.13s]
[2025-02-05T14:31:11.147+0000] {logging_mixin.py:188} INFO - 14:31:11
[2025-02-05T14:31:11.149+0000] {logging_mixin.py:188} INFO - 14:31:11  Finished running 1 table model in 0 hours 0 minutes and 0.24 seconds (0.24s).
[2025-02-05T14:31:11.161+0000] {logging_mixin.py:188} INFO - 14:31:11
[2025-02-05T14:31:11.162+0000] {logging_mixin.py:188} INFO - 14:31:11  Completed successfully
[2025-02-05T14:31:11.163+0000] {logging_mixin.py:188} INFO - 14:31:11
[2025-02-05T14:31:11.164+0000] {logging_mixin.py:188} INFO - 14:31:11  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-02-05T14:31:12.215+0000] {local.py:138} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v10.json is above dbt-ol supported version 7. This might cause errors.
[2025-02-05T14:31:12.272+0000] {local.py:494} INFO - Inlets: []
[2025-02-05T14:31:12.273+0000] {local.py:495} INFO - Outlets: [Dataset(uri='postgres://postgres-db:5432/localiza_dw.dm_sales.ad_hoc_summary_vendas', extra=None)]
[2025-02-05T14:31:12.274+0000] {local.py:595} INFO - Assigning inlets/outlets without DatasetAlias
[2025-02-05T14:31:12.275+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2025-02-05T14:31:12.318+0000] {dag.py:3954} INFO - Setting next_dagrun for dag_dbt-model_sales to 2025-02-05 04:00:00+00:00, run_after=2025-02-06 04:00:00+00:00
[2025-02-05T14:31:12.365+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-05T14:31:12.394+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_dbt-model_sales, task_id=model_sales.ad_hoc_summary_vendas_run, run_id=scheduled__2025-02-04T04:00:00+00:00, execution_date=20250204T040000, start_date=20250205T143106, end_date=20250205T143112
[2025-02-05T14:31:12.433+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-05T14:31:12.456+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-05T14:31:12.460+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-02-05T14:56:32.786+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-05T14:56:32.803+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_dbt-model_sales.model_sales.ad_hoc_summary_vendas_run scheduled__2025-02-04T04:00:00+00:00 [queued]>
[2025-02-05T14:56:32.811+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_dbt-model_sales.model_sales.ad_hoc_summary_vendas_run scheduled__2025-02-04T04:00:00+00:00 [queued]>
[2025-02-05T14:56:32.811+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-02-05T14:56:32.825+0000] {taskinstance.py:2330} INFO - Executing <Task(DbtRunLocalOperator): model_sales.ad_hoc_summary_vendas_run> on 2025-02-04 04:00:00+00:00
[2025-02-05T14:56:32.831+0000] {standard_task_runner.py:63} INFO - Started process 170 to run task
[2025-02-05T14:56:32.835+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_dbt-model_sales', 'model_sales.ad_hoc_summary_vendas_run', 'scheduled__2025-02-04T04:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/dag_dbt-model_sales.py', '--cfg-path', '/tmp/tmpsrs8_pp8']
[2025-02-05T14:56:32.836+0000] {standard_task_runner.py:91} INFO - Job 8: Subtask model_sales.ad_hoc_summary_vendas_run
[2025-02-05T14:56:32.890+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_dbt-model_sales.model_sales.ad_hoc_summary_vendas_run scheduled__2025-02-04T04:00:00+00:00 [running]> on host 7c7f80fafcb7
[2025-02-05T14:56:32.964+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_dbt-model_sales' AIRFLOW_CTX_TASK_ID='model_sales.ad_hoc_summary_vendas_run' AIRFLOW_CTX_EXECUTION_DATE='2025-02-04T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-04T04:00:00+00:00'
[2025-02-05T14:56:32.966+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-05T14:56:35.027+0000] {local.py:220} INFO - dbtRunner is available. Using dbtRunner for invoking dbt.
[2025-02-05T14:56:35.029+0000] {local.py:436} INFO - Cloning project to writable temp directory /tmp/tmpa39212_p from /usr/local/airflow/dags/dbt/localiza_dw
[2025-02-05T14:56:35.036+0000] {local.py:447} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/dag_dbt-model_sales__model_sales/target/partial_parse.msgpack
[2025-02-05T14:56:35.042+0000] {config.py:348} INFO - Profile caching is enable.
[2025-02-05T14:56:35.050+0000] {base.py:84} INFO - Using connection ID '***' for task execution.
[2025-02-05T14:56:35.051+0000] {config.py:328} INFO - Profile found in cache using profile: /tmp/cosmos/profile/47873256e3df2d6067f929148b3671caee72f33396318ef25aae33418849eddf/profiles.yml.
[2025-02-05T14:56:35.051+0000] {local.py:408} INFO - Trying to run dbtRunner with:
 ['run', '--vars', '{"PROD": P003,                                 "LIMITE_QUANTIDADE": 500,                                 "DATA_INICIO": 1999-01-01,                                 "DATA_FIM": 2025-02-05}', '--models', 'ad_hoc_summary_vendas', '--project-dir', '/tmp/tmpa39212_p', '--profiles-dir', '/tmp/cosmos/profile/47873256e3df2d6067f929148b3671caee72f33396318ef25aae33418849eddf', '--profile', '***', '--target', 'prod']
 in /tmp/tmpa39212_p
[2025-02-05T14:56:35.068+0000] {logging_mixin.py:188} INFO - 14:56:35  Running with dbt=1.6.1
[2025-02-05T14:56:35.166+0000] {logging_mixin.py:188} INFO - 14:56:35  Registered adapter: postgres=1.6.1
[2025-02-05T14:56:35.719+0000] {logging_mixin.py:188} INFO - 14:56:35  Found 4 models, 1 snapshot, 4 seeds, 5 tests, 3 sources, 0 exposures, 0 metrics, 349 macros, 0 groups, 0 semantic models
[2025-02-05T14:56:35.722+0000] {logging_mixin.py:188} INFO - 14:56:35
[2025-02-05T14:56:35.793+0000] {logging_mixin.py:188} INFO - 14:56:35  Concurrency: 1 threads (target='prod')
[2025-02-05T14:56:35.795+0000] {logging_mixin.py:188} INFO - 14:56:35
[2025-02-05T14:56:35.798+0000] {logging_mixin.py:188} INFO - 14:56:35  1 of 1 START sql table model dm_sales.ad_hoc_summary_vendas .................... [RUN]
[2025-02-05T14:56:35.894+0000] {logging_mixin.py:188} INFO - 14:56:35  1 of 1 OK created sql table model dm_sales.ad_hoc_summary_vendas ............... [SELECT 5 in 0.09s]
[2025-02-05T14:56:35.906+0000] {logging_mixin.py:188} INFO - 14:56:35
[2025-02-05T14:56:35.907+0000] {logging_mixin.py:188} INFO - 14:56:35  Finished running 1 table model in 0 hours 0 minutes and 0.18 seconds (0.18s).
[2025-02-05T14:56:35.915+0000] {logging_mixin.py:188} INFO - 14:56:35
[2025-02-05T14:56:35.917+0000] {logging_mixin.py:188} INFO - 14:56:35  Completed successfully
[2025-02-05T14:56:35.918+0000] {logging_mixin.py:188} INFO - 14:56:35
[2025-02-05T14:56:35.918+0000] {logging_mixin.py:188} INFO - 14:56:35  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-02-05T14:56:36.687+0000] {local.py:138} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v10.json is above dbt-ol supported version 7. This might cause errors.
[2025-02-05T14:56:36.720+0000] {local.py:494} INFO - Inlets: []
[2025-02-05T14:56:36.720+0000] {local.py:495} INFO - Outlets: [Dataset(uri='postgres://postgres-db:5432/localiza_dw.dm_sales.ad_hoc_summary_vendas', extra=None)]
[2025-02-05T14:56:36.721+0000] {local.py:595} INFO - Assigning inlets/outlets without DatasetAlias
[2025-02-05T14:56:36.722+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2025-02-05T14:56:36.743+0000] {dag.py:3954} INFO - Setting next_dagrun for dag_dbt-model_sales to 2025-02-05 04:00:00+00:00, run_after=2025-02-06 04:00:00+00:00
[2025-02-05T14:56:36.775+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-05T14:56:36.791+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_dbt-model_sales, task_id=model_sales.ad_hoc_summary_vendas_run, run_id=scheduled__2025-02-04T04:00:00+00:00, execution_date=20250204T040000, start_date=20250205T145632, end_date=20250205T145636
[2025-02-05T14:56:36.821+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-05T14:56:36.838+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-05T14:56:36.840+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-02-05T15:08:05.748+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-05T15:08:05.766+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_dbt-model_sales.model_sales.ad_hoc_summary_vendas_run scheduled__2025-02-04T04:00:00+00:00 [queued]>
[2025-02-05T15:08:05.776+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_dbt-model_sales.model_sales.ad_hoc_summary_vendas_run scheduled__2025-02-04T04:00:00+00:00 [queued]>
[2025-02-05T15:08:05.777+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-02-05T15:08:05.788+0000] {taskinstance.py:2330} INFO - Executing <Task(DbtRunLocalOperator): model_sales.ad_hoc_summary_vendas_run> on 2025-02-04 04:00:00+00:00
[2025-02-05T15:08:05.792+0000] {standard_task_runner.py:63} INFO - Started process 170 to run task
[2025-02-05T15:08:05.795+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_dbt-model_sales', 'model_sales.ad_hoc_summary_vendas_run', 'scheduled__2025-02-04T04:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/dag_dbt-model_sales.py', '--cfg-path', '/tmp/tmpxebpw29b']
[2025-02-05T15:08:05.797+0000] {standard_task_runner.py:91} INFO - Job 8: Subtask model_sales.ad_hoc_summary_vendas_run
[2025-02-05T15:08:05.844+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_dbt-model_sales.model_sales.ad_hoc_summary_vendas_run scheduled__2025-02-04T04:00:00+00:00 [running]> on host 5c897c3a2254
[2025-02-05T15:08:05.924+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_dbt-model_sales' AIRFLOW_CTX_TASK_ID='model_sales.ad_hoc_summary_vendas_run' AIRFLOW_CTX_EXECUTION_DATE='2025-02-04T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-04T04:00:00+00:00'
[2025-02-05T15:08:05.926+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-05T15:08:08.172+0000] {local.py:220} INFO - dbtRunner is available. Using dbtRunner for invoking dbt.
[2025-02-05T15:08:08.174+0000] {local.py:436} INFO - Cloning project to writable temp directory /tmp/tmpvppqwyq1 from /usr/local/airflow/dags/dbt/localiza_dw
[2025-02-05T15:08:08.180+0000] {local.py:447} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/dag_dbt-model_sales__model_sales/target/partial_parse.msgpack
[2025-02-05T15:08:08.187+0000] {config.py:348} INFO - Profile caching is enable.
[2025-02-05T15:08:08.198+0000] {base.py:84} INFO - Using connection ID '***' for task execution.
[2025-02-05T15:08:08.200+0000] {config.py:328} INFO - Profile found in cache using profile: /tmp/cosmos/profile/47873256e3df2d6067f929148b3671caee72f33396318ef25aae33418849eddf/profiles.yml.
[2025-02-05T15:08:08.200+0000] {local.py:408} INFO - Trying to run dbtRunner with:
 ['run', '--vars', '{"PROD": "P003", "LIMITE_QUANTIDADE": "500", "DATA_INICIO": "1999-01-01", "DATA_FIM": "2025-02-05"}', '--models', 'ad_hoc_summary_vendas', '--project-dir', '/tmp/tmpvppqwyq1', '--profiles-dir', '/tmp/cosmos/profile/47873256e3df2d6067f929148b3671caee72f33396318ef25aae33418849eddf', '--profile', '***', '--target', 'prod']
 in /tmp/tmpvppqwyq1
[2025-02-05T15:08:08.219+0000] {logging_mixin.py:188} INFO - 15:08:08  Running with dbt=1.6.1
[2025-02-05T15:08:08.327+0000] {logging_mixin.py:188} INFO - 15:08:08  Registered adapter: postgres=1.6.1
[2025-02-05T15:08:08.884+0000] {logging_mixin.py:188} INFO - 15:08:08  Found 4 models, 1 snapshot, 4 seeds, 5 tests, 3 sources, 0 exposures, 0 metrics, 349 macros, 0 groups, 0 semantic models
[2025-02-05T15:08:08.886+0000] {logging_mixin.py:188} INFO - 15:08:08
[2025-02-05T15:08:08.959+0000] {logging_mixin.py:188} INFO - 15:08:08  Concurrency: 1 threads (target='prod')
[2025-02-05T15:08:08.960+0000] {logging_mixin.py:188} INFO - 15:08:08
[2025-02-05T15:08:08.964+0000] {logging_mixin.py:188} INFO - 15:08:08  1 of 1 START sql table model dm_sales.ad_hoc_summary_vendas .................... [RUN]
[2025-02-05T15:08:09.058+0000] {logging_mixin.py:188} INFO - 15:08:09  1 of 1 OK created sql table model dm_sales.ad_hoc_summary_vendas ............... [SELECT 5 in 0.09s]
[2025-02-05T15:08:09.070+0000] {logging_mixin.py:188} INFO - 15:08:09
[2025-02-05T15:08:09.071+0000] {logging_mixin.py:188} INFO - 15:08:09  Finished running 1 table model in 0 hours 0 minutes and 0.18 seconds (0.18s).
[2025-02-05T15:08:09.079+0000] {logging_mixin.py:188} INFO - 15:08:09
[2025-02-05T15:08:09.080+0000] {logging_mixin.py:188} INFO - 15:08:09  Completed successfully
[2025-02-05T15:08:09.081+0000] {logging_mixin.py:188} INFO - 15:08:09
[2025-02-05T15:08:09.082+0000] {logging_mixin.py:188} INFO - 15:08:09  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-02-05T15:08:10.064+0000] {local.py:138} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v10.json is above dbt-ol supported version 7. This might cause errors.
[2025-02-05T15:08:10.099+0000] {local.py:494} INFO - Inlets: []
[2025-02-05T15:08:10.100+0000] {local.py:495} INFO - Outlets: [Dataset(uri='postgres://postgres-db:5432/localiza_dw.dm_sales.ad_hoc_summary_vendas', extra=None)]
[2025-02-05T15:08:10.100+0000] {local.py:595} INFO - Assigning inlets/outlets without DatasetAlias
[2025-02-05T15:08:10.101+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2025-02-05T15:08:10.125+0000] {dag.py:3954} INFO - Setting next_dagrun for dag_dbt-model_sales to 2025-02-05 04:00:00+00:00, run_after=2025-02-06 04:00:00+00:00
[2025-02-05T15:08:10.154+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-05T15:08:10.176+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_dbt-model_sales, task_id=model_sales.ad_hoc_summary_vendas_run, run_id=scheduled__2025-02-04T04:00:00+00:00, execution_date=20250204T040000, start_date=20250205T150805, end_date=20250205T150810
[2025-02-05T15:08:10.206+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-05T15:08:10.226+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-05T15:08:10.228+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-02-05T16:46:13.006+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-05T16:46:13.021+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_dbt-model_sales.model_sales.ad_hoc_summary_vendas_run scheduled__2025-02-04T04:00:00+00:00 [queued]>
[2025-02-05T16:46:13.028+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_dbt-model_sales.model_sales.ad_hoc_summary_vendas_run scheduled__2025-02-04T04:00:00+00:00 [queued]>
[2025-02-05T16:46:13.029+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-02-05T16:46:13.040+0000] {taskinstance.py:2330} INFO - Executing <Task(DbtRunLocalOperator): model_sales.ad_hoc_summary_vendas_run> on 2025-02-04 04:00:00+00:00
[2025-02-05T16:46:13.044+0000] {standard_task_runner.py:63} INFO - Started process 193 to run task
[2025-02-05T16:46:13.047+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_dbt-model_sales', 'model_sales.ad_hoc_summary_vendas_run', 'scheduled__2025-02-04T04:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/dag_dbt-model_sales.py', '--cfg-path', '/tmp/tmpplso9ddg']
[2025-02-05T16:46:13.049+0000] {standard_task_runner.py:91} INFO - Job 9: Subtask model_sales.ad_hoc_summary_vendas_run
[2025-02-05T16:46:13.095+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_dbt-model_sales.model_sales.ad_hoc_summary_vendas_run scheduled__2025-02-04T04:00:00+00:00 [running]> on host 584eb3626689
[2025-02-05T16:46:13.164+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_dbt-model_sales' AIRFLOW_CTX_TASK_ID='model_sales.ad_hoc_summary_vendas_run' AIRFLOW_CTX_EXECUTION_DATE='2025-02-04T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-04T04:00:00+00:00'
[2025-02-05T16:46:13.166+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-05T16:46:15.211+0000] {local.py:220} INFO - dbtRunner is available. Using dbtRunner for invoking dbt.
[2025-02-05T16:46:15.213+0000] {local.py:436} INFO - Cloning project to writable temp directory /tmp/tmp6khigz0e from /usr/local/airflow/dags/dbt/localiza_dw
[2025-02-05T16:46:15.221+0000] {local.py:447} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/dag_dbt-model_sales__model_sales/target/partial_parse.msgpack
[2025-02-05T16:46:15.227+0000] {config.py:348} INFO - Profile caching is enable.
[2025-02-05T16:46:15.234+0000] {base.py:84} INFO - Using connection ID '***' for task execution.
[2025-02-05T16:46:15.236+0000] {config.py:328} INFO - Profile found in cache using profile: /tmp/cosmos/profile/47873256e3df2d6067f929148b3671caee72f33396318ef25aae33418849eddf/profiles.yml.
[2025-02-05T16:46:15.237+0000] {local.py:408} INFO - Trying to run dbtRunner with:
 ['run', '--vars', '{"PROD": "P003", "LIMITE_QUANTIDADE": "500", "DATA_INICIO": "1999-01-01", "DATA_FIM": "2025-02-05"}', '--models', 'ad_hoc_summary_vendas', '--project-dir', '/tmp/tmp6khigz0e', '--profiles-dir', '/tmp/cosmos/profile/47873256e3df2d6067f929148b3671caee72f33396318ef25aae33418849eddf', '--profile', '***', '--target', 'prod']
 in /tmp/tmp6khigz0e
[2025-02-05T16:46:15.252+0000] {logging_mixin.py:188} INFO - 16:46:15  Running with dbt=1.6.1
[2025-02-05T16:46:15.358+0000] {logging_mixin.py:188} INFO - 16:46:15  Registered adapter: postgres=1.6.1
[2025-02-05T16:46:16.039+0000] {logging_mixin.py:188} INFO - 16:46:16  Found 5 models, 1 snapshot, 4 seeds, 5 tests, 3 sources, 0 exposures, 0 metrics, 349 macros, 0 groups, 0 semantic models
[2025-02-05T16:46:16.042+0000] {logging_mixin.py:188} INFO - 16:46:16
[2025-02-05T16:46:16.129+0000] {logging_mixin.py:188} INFO - 16:46:16  Concurrency: 1 threads (target='prod')
[2025-02-05T16:46:16.131+0000] {logging_mixin.py:188} INFO - 16:46:16
[2025-02-05T16:46:16.136+0000] {logging_mixin.py:188} INFO - 16:46:16  1 of 1 START sql table model dm_sales.ad_hoc_summary_vendas .................... [RUN]
[2025-02-05T16:46:16.238+0000] {logging_mixin.py:188} INFO - 16:46:16  1 of 1 OK created sql table model dm_sales.ad_hoc_summary_vendas ............... [SELECT 5 in 0.10s]
[2025-02-05T16:46:16.250+0000] {logging_mixin.py:188} INFO - 16:46:16
[2025-02-05T16:46:16.251+0000] {logging_mixin.py:188} INFO - 16:46:16  Finished running 1 table model in 0 hours 0 minutes and 0.21 seconds (0.21s).
[2025-02-05T16:46:16.260+0000] {logging_mixin.py:188} INFO - 16:46:16
[2025-02-05T16:46:16.262+0000] {logging_mixin.py:188} INFO - 16:46:16  Completed successfully
[2025-02-05T16:46:16.263+0000] {logging_mixin.py:188} INFO - 16:46:16
[2025-02-05T16:46:16.264+0000] {logging_mixin.py:188} INFO - 16:46:16  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-02-05T16:46:17.205+0000] {local.py:138} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v10.json is above dbt-ol supported version 7. This might cause errors.
[2025-02-05T16:46:17.254+0000] {local.py:494} INFO - Inlets: []
[2025-02-05T16:46:17.254+0000] {local.py:495} INFO - Outlets: [Dataset(uri='postgres://postgres-db:5432/localiza_dw.dm_sales.ad_hoc_summary_vendas', extra=None)]
[2025-02-05T16:46:17.255+0000] {local.py:595} INFO - Assigning inlets/outlets without DatasetAlias
[2025-02-05T16:46:17.257+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2025-02-05T16:46:17.283+0000] {dag.py:3954} INFO - Setting next_dagrun for dag_dbt-model_sales to 2025-02-05 04:00:00+00:00, run_after=2025-02-06 04:00:00+00:00
[2025-02-05T16:46:17.315+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-05T16:46:17.334+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_dbt-model_sales, task_id=model_sales.ad_hoc_summary_vendas_run, run_id=scheduled__2025-02-04T04:00:00+00:00, execution_date=20250204T040000, start_date=20250205T164613, end_date=20250205T164617
[2025-02-05T16:46:17.382+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-05T16:46:17.404+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-05T16:46:17.406+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
